toonz/sources/image/pli/tiio_pli.cpp

```diff
@@ -541,7 +550,13 @@ solo nel costruttore)
     PliTag *tag = new PrecisionScaleTag(precisionScale);
     tags.push_back((PliObjectTag *)tag);
   }
-
+  // Store the auto close tolerance
+  {
+    int tolerance =
+        (int)((roundf(tempVecImg->getAutocloseTolerance() * 100) / 100) * 1000);
+    PliTag *tag = new AutoCloseToleranceTag(tolerance);
+    tags.push_back((PliObjectTag *)tag);
+  }
```

Comment:
Sorry if I'm wrong, but skipping this scope (if the parameter is unchanged from the default) can save PLI compatible with older versions?

Reply:
Yes.  That's what I am building in.